common/tools/dev-tool/src/commands/samples/run.ts

```diff
@@ -69,6 +69,10 @@ export default leafCommand(commandInfo, async (options) => {
     log.warn("To skip samples in live tests pipelines, disable them using the package's tests.yml");
   }
 
+  if (packageJson.name.includes("-arm")) {
```

Comment:
```suggestion
  if (packageJson.name.includes("arm-")) {
```

I assume you're looking for prefix `arm-` instead of `-arm`?

Reply:
Yes I am, thanks 😄